Intersect.Server/Entities/Player.cs

```diff
@@ -3605,6 +3605,8 @@ public void CraftItem(Guid id)
                     invbackup.Add(item.Clone());
                 }
 
+                var mCraftItem = CraftBase.Get(id);
```

Comment:
Just `craftItem` -- `m` is used for member variables on a class, this is only local.